### PR TITLE
Change number display to name

### PR DESCRIPTION
**Pull request by dewin017**

### CODE REVIEW

File: app/app.iml [hunk 1]
```diff
@@ -64,14 +64,6 @@
       <sourceFolder url="file://$MODULE_DIR$/src/main/jni" isTestSource="false" />
```
Comment (rohands) [inline app/app.iml]: @kisaacs By convention, we refrain from committing auto generated stub (files) like app/app.iml

**Comment by sandarumk** [comment]
Actually you are quite right Rohan. @kisaacs @dewin017  Please read the wiki on submitting issues, submitting PRs and reviewing them.

**Comment by kisaacs** [comment]
Again, sorry about that. I didn't realize such files came with the repository. You can revert and we'll redo.

This was for GHC OSD and we asked @MeepyMay about making changes without going through the reporting process given the README.

**Comment by sandarumk** [comment]
It is okay if you skip the confimation process since it is OSD. But reporting the issue first and submitting the PR will make life easier for everyone. Since it is an open source project there are other people rather than the ones at OSD contributing parallel as well. As you can see there are few remaining PRs from other students and few active issues. In future, please go through the repository documentation well and guidelines and standards before merging any pull requests.

**Comment by kisaacs** [comment]
Yes, we ran into some problems due to the DDOS making github unreachable which put things in an even stranger state. In retrospect, I think we should have done everything in an OSD-specific branch so the problems would affect less. I'll recommend that for next year. I'm traveling currently and will contact the students when I return to try to follow the whole cycle for the few issues we believe we found (the toolbar Rohan pointed us at and another that hasn't been PR'd). Once again, I really appreciate your patience with the matter.

**Comment by sandarumk** [comment]
Merge reverted.

**Pull request closed without merging**